### ISSUE TITLE
500 error when loading a page with the latest entries plugin

### ISSUE THREAD

**Issue by stefanfoulis** [aldryn/aldryn-newsblog#218]
I get the following error on a project where haystack is not installed:

```
Environment:


Request Method: GET
Request URL: http://localhost:8000/en/admin/cms/page/1/en/publish/

Django Version: 1.7.8
Python Version: 2.7.8
Installed Applications:
('djangocms_admin_style',
 'djangocms_text_ckeditor',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.admin',
 'django.contrib.sites',
 'django.contrib.sitemaps',
 'django.contrib.staticfiles',
 'django.contrib.messages',
 'cms',
 'menus',
 'sekizai',
 'treebeard',
 'djangocms_style',
 'djangocms_column',
 'djangocms_file',
 'djangocms_flash',
 'djangocms_googlemap',
 'djangocms_inherit',
 'djangocms_link',
 'djangocms_picture',
 'djangocms_teaser',
 'djangocms_video',
 'reversion',
 'mysite',
 'polls',
 'polls_plugin',
 'aldryn_apphooks_config',
 'aldryn_boilerplates',
 'aldryn_categories',
 'aldryn_newsblog',
 'aldryn_people',
 'aldryn_reversion',
 'easy_thumbnails',
 'filer',
 'parler',
 'sortedm2m',
 'taggit')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.doc.XViewMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'cms.middleware.user.CurrentUserMiddleware',
 'cms.middleware.page.CurrentPageMiddleware',
 'cms.middleware.toolbar.ToolbarMiddleware',
 'cms.middleware.language.LanguageCookieMiddleware')


Traceback:
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  105.                     response = view_func(request, *args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  52.         response = view_func(request, *args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
  206.             return view(request, *args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/db/transaction.py" in inner
  394.                 return func(*args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/reversion/revisions.py" in do_revision_context
  297.                 return func(*args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/cms/admin/pageadmin.py" in publish_page
  956.             published = page.publish(language)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/cms/models/pagemodel.py" in publish
  608.             self._copy_contents(public_page, language)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/cms/models/pagemodel.py" in _copy_contents
  308.                 copy_plugins_to(plugins, ph, no_signals=True)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/cms/utils/copy_plugins.py" in copy_plugins_to
  32.             new_instance.post_copy(old_plugin, plugins_ziplist)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/djangocms_text_ckeditor/models.py" in post_copy
  73.         self.save()
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/djangocms_text_ckeditor/models.py" in save
  44.         super(AbstractText, self).save(*args, **kwargs)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/cms/models/pluginmodel.py" in save
  238.         super(CMSPlugin, self).save()
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/db/models/base.py" in save
  589.                        force_update=force_update, update_fields=update_fields)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/db/models/base.py" in save_base
  626.                                    update_fields=update_fields, raw=raw, using=using)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/lib/python2.7/site-packages/django/dispatch/dispatcher.py" in send
  198.             response = receiver(signal=self, sender=sender, **named)
File "/Users/stefanfoulis/Coding/Tutorials/django-cms-tutorial-code/env/src/aldryn-newsblog/aldryn_newsblog/models.py" in update_seach_index
  523.         placeholder = instance._placeholder_cache

Exception Type: AttributeError at /en/admin/cms/page/1/en/publish/
Exception Value: 'Text' object has no attribute '_placeholder_cache'
```

**Comment by stefanfoulis** [comment]
I can't find any reference to a CMSPlugin Model instance having a ``_placeholder_cache`` anywhere in the django-cms code. I'll submit it a pull request to make it use ``instance.placeholder`` instead.

**Comment by mkoistinen** [comment]
Closed by #220.
